pyiceberg/table/snapshots.py

```diff
@@ -65,6 +90,25 @@ def __init__(self, operation: Operation, **data: Any) -> None:
         super().__init__(operation=operation, **data)
         self._additional_properties = data
 
+    def __getitem__(self, __key: str) -> Optional[Any]:  # type: ignore
+        """Return a key as it is a map."""
+        if __key == 'operation':
```

Comment:
Should this be `OPERATION`?

Reply:
It seems to be lower-case here:
![image](https://github.com/apache/iceberg-python/assets/1134248/706c04e2-0f2e-4824-9ddb-8485ed9fcd3a)

I can make it case-insensitive

Reply:
I meant that we have a constant defined and don't need to embed the string. Not a big deal.